docs/assets/js/main.js

```diff
@@ -35,6 +37,23 @@ if ( multiselectDom ) {
 Expandable.init();
 Table.init();
 
+// Transition example code.
+const moveTransitionExample = document.querySelector( '.example-box.u-move-transition' );
+if ( moveTransitionExample !== null ) {
+  const moveTransition = new MoveTransition( moveTransitionExample ).init();
+  moveTransitionExample.addEventListener( 'click', () => {
+    moveTransition.moveRight();
+  } );
+}
+
+const alphaTransitionExample = document.querySelector( '.example-box.u-alpha-transition' );
+if ( alphaTransitionExample !== null ) {
+  const alphaTransition = new AlphaTransition( alphaTransitionExample ).init();
+  alphaTransitionExample.addEventListener( 'click', () => {
+    alphaTransition.fadeOut();
```

Comment:
To make it easier for devs to inspect the element and repeatedly click it, can we have the example fade back in?

```suggestion
    alphaTransition.fadeOut();
    setTimeout( () => {
      alphaTransition.fadeIn();
    }, 2000 );
```

Reply:
I'll add that for the move one too.